src/components/QuizGameApi/index.jsx

```diff
@@ -0,0 +1,22 @@
+// first line to rendomlly picking a question
+// we fetch those questions from opentdb website. we can pick the type of questions,the amount and the dificulty.
+// here we have a 10 questions as an amount, nature as a category ,easy, middle and hard for the difficulty and multiple choices as a type.
+const shuffleArray=(array) =>{
+  for (let i = array.length - 1; i > 0; i--) {
+    let j = Math.floor(Math.random() * (i + 1));
+    [array[i], array[j]] = [array[j], array[i]];
+  }
+}
```

Comment:
I already added this to `components/memorygame` folder as `shuffle.jsx` . You can move it back under `src/components` and import it.

Reply:
PTAL